src/features/brandsCreator.js

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const BASE_URL = 'http://localhost:5000';
+
+export const allBrands = createAsyncThunk('brand/allBrands', async () => {
```

Comment:
please check the thunks naming in official docs https://redux-toolkit.js.org/api/createAsyncThunk

Reply:
fix around all the files